dotCMS/src/main/java/com/dotcms/integritycheckers/IntegrityUtil.java

```diff
@@ -320,14 +337,14 @@ public void dropTempTables(final String endpointId) throws DotDataException {
         try {
             IntegrityType[] types = IntegrityType.values();
             for (IntegrityType integrityType : types) {
-                final String tempTableName = getTempTableName(endpointId, integrityType);
-
-                if (doesTableExist(tempTableName)) {
-                    dc.executeStatement("truncate table " + tempTableName);
-                    dc.executeStatement("drop table " + tempTableName);
-                }else if (DbConnectionFactory.isMySql()){
-                    dc.executeStatement("drop table if exists " + tempTableName);
-                }
+            	for(String tempTableName : integrityType.getIntegrityChecker().getTempTableNames(endpointId)) {
```

Comment:
I think this part could be useful to have some logging, even for an else if the 

```}else if (DbConnectionFactory.isMySql()){```

Is false